src/core/config/upstream.rs

```diff
@@ -28,7 +28,13 @@ impl Default for Batch {
         }
     }
 }
-
+#[derive(Serialize, Deserialize, PartialEq, Eq, Clone, Debug, schemars::JsonSchema, MergeRight)]
+#[serde(rename_all = "camelCase", default)]
```

Comment:
should not use default here, we can do that wherever we use this struct as a field

Reply:
@ssddOnTop On removing **default** doesn't the user have to specify **enable** as now its a required field, it makes sense if we remove **_enable_** though than only **_size_** is the attribute present and it should be mentioned explicitly.
![image](https://github.com/tailcallhq/graphql-runtime/assets/81297089/64818cc4-e4d7-4152-8447-8a44686fc8b5)